### PR TITLE
Development

### PR DESCRIPTION
**Pull request by alanluqman**
## Description 📃 
In this Pull Request, my partner @ogagaoghene and I have created this website for who likes to join SpaceX Missions and reserve a desired Rocket, this website is built with `JavaScript`, `React Redux`, also we have created test files for these three pages (`Rockets`, `Missions`, `My profile`) with `Jest.js`

### CODE REVIEW

File: src/test/mission.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import store from '../redux/configureStore';
+import Missions from '../pages/Missions';
+
+describe('Missions page component', () => {
+  afterEach(() => {
+    act(() => store.dispatch({
+      type: 'spacetravelhub/missions/fetch',
+      payload: [],
+    }));
+  });
```
Comment (uwadonat) [inline src/test/mission.test.js:14]: - [ ] One of your tests is failing
Screenshot from 2022-08-26 10-55-11

File: package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "jest": "^28.1.3",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
+    "react-render": "^4.0.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
```
Comment (uwadonat) [inline package.json:18]: - [ ] You are supposed to have `Redux Logger` in your `package.json` file.

File: src/pages/MyProfile.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="container">
+      <div className="display-section">
+        <h1>My Missions</h1>
+        <ul>
+          {reservedMissions.map((element) => <li key={element.id}>{element.mission_name}</li>)}
+        </ul>
+      </div>
+      <div className="display-section">
+        <h1>My Rockets</h1>
+        <ul>
```
Comment (uwadonat) [inline src/pages/MyProfile.js:20]: **Optional:**
- [ ] The My Profile section can be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved).

File: README.md [hunk 4]
```diff
@@ -1 +1,51 @@
...
+## Get Started
+To get a local copy of this calculator, Please follow these simple example steps.
+1. Open Git Bash.
+2. Change the current working directory to the location where you want the cloned directory. 
+ Ex: `C:User\your computer name\ cd <<your folder you want to clone to>>`
+4. Copy/Past this command :  `git clone https://github.com/alanluqman/spacetravelhub.git`  then press Enter
+5. Run this command `code .` to open this folder in VScode
+6. Open terminal and run this command `npm install`
+7. after finishing step 5, run this command `npm start` to run the project
+
```
Comment (uwadonat) [inline README.md:22]: - [ ] Let's also add instructions to test project. It will make your project more easy to use.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Very descriptive pull request message ✔️
- No linter errors ✔️
- Your design looks amazing ✔️
approval


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/mission.test.js (hunk 1 above)
Comment (alanluqman) [reply src/test/mission.test.js:14]: Dear @uwadonat, actually I don't know why its failed from you but everything is correct in my computer. this is a screenshot of my run test >
test-spaceX

**Comment by alanluqman** [comment]
Dear @uwadonat Thanks for your kindness review. 👍🏻

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alanluqman** [comment]
Dear @CollinsTatang thanks for you professional review 👍🏻

**Comment by ogagaoghene** [comment]
Thanks for your professional code review

**Pull request merged**